src/libs/actions/IOU.js

```diff
@@ -298,8 +298,10 @@ function buildOnyxDataForMoneyRequest(
  * @param {Number} payeeAccountID
  * @param {Object} participant
  * @param {String} comment
+ * @param {Object} [receipt]
+ *
  */
-function requestMoney(report, amount, currency, payeeEmail, payeeAccountID, participant, comment) {
+function requestMoney(report, amount, currency, payeeEmail, payeeAccountID, participant, comment, receipt = undefined) {
```

Comment:
nit: default value if undefined is not required.

Reply:
I'll leave this as `undefined` to be more explicit.